src/components/PlanningTool.jsx

```diff
@@ -111,7 +120,7 @@ class PlanningTool extends Component {
 
     const group = groups.filter(g => g.show)[newGroupOrder]
 
-    this.addUndoItem(items.find(i => i.id === itemId))
+    this.addUndoItem(items.find(i => i.id === itemId), 'item-edit', {}, true)
```

Comment:
It would be nice to use constants instead of strings (e.g. `Constants.itemEdit`).
Same for other ones below.

You can see an example here: https://github.com/kbss-cvut/s-forms/blob/master/src/constants/Constants.js

Reply:
✅